examples/initiator/operators_data/operators_info.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
are we adding another file? if this is used now instead of another lets remove the previous.